js/about/contributionStatement.js

```diff
@@ -285,6 +308,17 @@ class ContributionStatement extends ImmutableComponent {
   }
 
   ContributionStatementDetailTable (page, pageIdx, totalPages) {
+    const imgStyle = StyleSheet.create({
```

Comment:
same question as above: why not keep this style with the others as before (`styles.verified`) rather than creating it here?